snap/snapcraft.yaml

```diff
@@ -24,9 +24,23 @@ platforms:
     build-on: [arm64]
     build-for: [arm64]
 
+slots:
+  custom-cpu-latency-dev:
+    interface: custom-device
+    custom-device: cpu-dma-latency
+    devices:
+      - /dev/cpu_dma_latency
```

Comment:
Need to drop this as the access can soon be granted via cpu-control; see https://github.com/canonical/rt-tests-snap/issues/21#issuecomment-2211105632